kong.conf.default

```diff
@@ -29,6 +29,17 @@
                                  # Each Kong process must have a separate
                                  # working directory.
 
+#port_map =                      # When running Kong behind layer 4 port mapping
```

Comment:
Another point, it would be better to call this setting `port_maps` because it can contain an array of values (csv based).

Reply:
Ok, I pluralized it. We btw. have `proxy_listen` not `proxy_listeners`.

Reply:
Good point. My bad.